packages/node_modules/@node-red/editor-client/src/js/ui/view-tools.js

```diff
@@ -870,6 +1031,8 @@ RED.view.tools = (function() {
             RED.actions.add("core:wire-series-of-nodes", function() { wireSeriesOfNodes() })
             RED.actions.add("core:wire-node-to-multiple", function() { wireNodeToMultiple() })
 
+            RED.actions.add("core:split-wire-with-link-nodes",function(){splitWiresWithLinkNodes(RED.view.selectedLinks.length() ? RED.view.selectedLinks.toArray() : null);});
```

Comment:
```suggestion
            RED.actions.add("core:split-wire-with-link-nodes",function() { splitWiresWithLinkNodes() });
```
It looks like `splitWiresWithLinkNodes` will already defer to the selected links if nothing is passed in - so no need to get the selected wires at this point.

Reply:
Done